crux-bench/src/crux/bench.clj

```diff
@@ -54,24 +78,27 @@
     (log/infof "finished bench-ns '%s'." bench-ns)
 
     (doseq [result @*!bench-results*]
-      (println (json/write-str result)))))
+      (println (json/write-str result)))
+    (format-and-post-results-to-slack @*!bench-results*)))
```

Comment:
couple of things here:
* let's deref the atom once and let-bind it - it's not going to change in-between, here, but good practice regardless
* similar to the `and` comment, above - something like `(post-to-slack (->> results (map ->slack-message) (str/join ...))`?